libs/entry-components/search-filter/entry-search-filter.module.ts

```diff
@@ -14,6 +14,7 @@ import { DateTimeSearchFilterComponent } from './date-time/date-time-search-filt
 import { MatDatepickerModule } from '@angular/material/datepicker';
 import { MatFormFieldModule } from '@angular/material/form-field';
 import { DateSearchFilterComponent } from './date/date-search-filter.component';
+import { EntryValidationModule } from '../validation/entry-validation.module';

```

Comment:
Should be ```import { EntryValidationModule } from '@enigmatry/entry-components/validation';```

Reply:
Paths inside module can be referenced with local path (./)
Paths outside from other modules, must be referenced with @enigmatry/entry-components/...